migration/migration/015-tracing-compression.sql

```diff
@@ -0,0 +1,99 @@
+
+-- Fix compression on trace tables
+DO $block$
+DECLARE
+    _is_timescaledb_installed boolean = false;
+    _is_timescaledb_oss boolean = true;
+    _timescaledb_major_version int;
+    _timescaledb_minor_version int;
+    _is_compression_available boolean = false;
+    _is_multinode boolean = false;
+    _saved_search_path text;
+BEGIN
+    _is_timescaledb_installed = _prom_catalog.is_timescaledb_installed();
+
+    IF _is_timescaledb_installed THEN
+        _timescaledb_major_version = _prom_catalog.get_timescale_major_version();
+        _timescaledb_minor_version = _prom_catalog.get_timescale_minor_version();
+
+        _is_compression_available = CASE
+            WHEN _timescaledb_major_version >= 2 THEN true
+            WHEN _timescaledb_major_version = 1 and _timescaledb_minor_version >= 5 THEN true
+            ELSE false
+        END;
+
+        _is_multinode = _prom_catalog.is_multinode();
+
+        IF _timescaledb_major_version >= 2 THEN
+            _is_timescaledb_oss = (current_setting('timescaledb.license') = 'apache');
+        ELSE
+            _is_timescaledb_oss = (SELECT edition = 'apache' FROM timescaledb_information.license);
+        END IF;
+    END IF;
+
+    IF _is_timescaledb_installed THEN
+        IF _is_multinode THEN
+            --need to clear the search path while creating distributed
+            --hypertables because otherwise the datanodes don't find
+            --the right column types since type names are not schema
+            --qualified if in search path.
+            _saved_search_path := current_setting('search_path');
+            SET search_path = pg_temp;
```

Comment:
can we just always do this and avoid the if _is_multinode logic?

Reply:
yes